src/expo.ts

```diff
@@ -112,3 +185,13 @@ export function projectDeepLink(project: ProjectInfo, channel?: string): string
 
   return url.toString();
 }
+
+export function getBuildLogsUrl(build: BuildInfo): string {
+  const { project } = build;
+  const path = project
+    ? `/accounts/${project.ownerAccount.name}/projects/${project.slug}/builds/${build.id}`
+    : `/builds/${build.id}`;
+
+  const url = new URL(path, 'https://expo.dev');
+  return url.toString();
+}
```

Comment:
Some function was copied from eas-cli package

Reply:
Seems great for now, if you can add a comment with the permalink to the part that you copied, that would be great. Optionally something like:
```suggestion
// TODO: reuse this function from the original source
// see: <github permalink>
export function getBuildLogsUrl(build: BuildInfo): string {
  const { project } = build;
  const path = project
    ? `/accounts/${project.ownerAccount.name}/projects/${project.slug}/builds/${build.id}`
    : `/builds/${build.id}`;

  const url = new URL(path, 'https://expo.dev');
  return url.toString();
}
```